### PR TITLE
Updated injected_node.wrapped_sql for data tests

### PR DESCRIPTION
**Pull request by mikaelene**
The data tests feature uses in-code-sql that does not support all databases. This is a fix for supporting more databases and custom adapters. 

This should not effect currently supported databases in any way.

ref conversation on dbt slack: https://getdbt.slack.com/archives/C50NEBJGG/p1565962661033700

### CODE REVIEW

**Comment by drewbanin** [comment]
Hey @mikaelene, thanks for the PR! I think we'll be able to slip this in for the 0.14.1 release. Let me kick off the tests here - this will be ready to merge when they pass.

Thanks also for filling out the CLA. You're the first person to make a PR since we added it - glad to see it's working well :D 

@cla-bot check

**Comment by drewbanin** [comment]
Looks like there's a formatting issue here:
```
core/dbt/compilation.py:137:80: E501 line too long (82 > 79 characters)
```

You can run 
```
docker-compose run test tox -e flake8
```

locally to run the flake8 linter. I'll kick these tests off again once that's updated!

**Comment by mikaelene** [comment]
Hi, I've updated now to support flake8

**Comment by mikaelene** [comment]
I am using another pc today and the cla-bot apparently didn't get my cla-email. I've updated my git-profile. Do I need to do a "dummy"-commit or something?

**Comment by drewbanin** [comment]
Thanks @mikaelene! You can ignore the cla issue - I see your signature... will check out why this bot is misbehaving offline.

Just kicked the tests off - will merge this when they pass :)

**Comment by drewbanin** [comment]
oops, looks like flake8 is still complaining, try changing the code to something like:
```
                injected_node.wrapped_sql = (
                    "select count(*) as errors "
                    "from (\n{test_sql}\n) sbq").format(
                        test_sql=injected_node.injected_sql)
```

**Comment by drewbanin** [comment]
hey @mikaelene - are you using Windows? It looks like every line in this file has changed - is that a line ending thing?

**Comment by mikaelene** [comment]
Crap! 

I didn't see that. I am using both windows and mac and store my project on Dropbox. That might be the problem. Let me try again.

**Comment by mikaelene** [comment]
Looks better now right?

**Comment by drewbanin** [comment]
I'm still seeing +252 -251 :/

Screen Shot 2019-08-20 at 10 07 28 AM

**Comment by mikaelene** [comment]
Still learning to git :-(. How about now?

**Comment by drewbanin** [comment]
Looks great, thanks @mikaelene! Kicking off the tests here - will merge this when they pass :)

**Comment by drewbanin** [comment]
merging! Thanks for your contribution @mikaelene :)

**Comment by mikaelene** [comment]
Wow. Finally got it right. :-)
